index.html

```diff
@@ -10,7 +10,7 @@
   </head>
 
   <body>
-    <h1>Git Playground</h1>
+    Git Playground</h1>
```

Comment:
Hey, you forgot the open tag on your html file. Make sure to fix it before pushing your code please.